pytest/lib/cluster.py

```diff
@@ -636,10 +645,6 @@ def spin_up_node(config,
     is_local = config['local']
 
     args = make_boot_nodes_arg(boot_node)
-    logger.info("Starting node %s %s" %
-                (ordinal,
-                 ('with ' + '='.join(args) if args else 'as BOOT NODE')))
-
```

Comment:
Removing this means that we no longer see the ordinal in the logs and also that we don’t get this line when doing remote run. The second point isn’t maybe that crucial since at this point I doubt tests can actually run with remote nodes, but ordinal is something useful in the log.

Reply:
ordinal is printed immediately after the arguments:

```
[2022-01-18 20:14:22] INFO: Starting node: /var/lib/buildkite-agent/builds/buildkite-i-00824bbff719e81c2-1/nearprotocol/nearcore/target/debug/neard-1.23.1-c0551c84ba9f54509abe05cb1291dbeb33cbd85d --home /tmp/near/backward/test0 run
[2022-01-18 20:14:25] INFO: node 0 started
[2022-01-18 20:14:25] INFO: Starting node: /var/lib/buildkite-agent/builds/buildkite-i-00824bbff719e81c2-1/nearprotocol/nearcore/target/debug/neard-nagisa-pytest-more-info --home /tmp/near/backward/test1 run --boot-nodes He7QeRuwizNEhBioYG3u4DZ8jWXyETiyNzFD3MkTjDMf@127.0.0.1:24577
[2022-01-18 20:14:30] INFO: node 1 started
```

Do you think we need to put the ordinal in the message with arguments still?

Reply:
My goal was to make it feasible to mostly copy and paste the command locally. Trying to post-process the invocation to e.g. relativize the paths doesn't feel super straightforward, to be honest.

That said my change has another problem I just noticed – it drops the print for GCloudNode. I don't really want to fiddle with this further so I'll just revert.

Reply:
> My goal was to make it feasible to mostly copy and paste the command locally.
I’ve never needed that feature but it makes sense.

> That said my change has another problem I just noticed – it drops the print for GCloudNode.

Yeah, that’s what I meant about remote runs. We could choose to ignore this though since we don’t use that feature and it may very well be that it doesn’t actually work anyway.